code/game/objects/structures/roguetown/doorbell.dm

```diff
@@ -0,0 +1,34 @@
+/obj/structure/doorbell
+	name = "bell"
+	desc = "This bell looks big enough to be heard inside the building."
+	icon = 'icons/roguetown/misc/structure.dmi'
+	icon_state = "bell"
+	density = FALSE
+	max_integrity = 0
+	anchored = TRUE
+	var/bell_name = "doorbell"
+	var/area/bell_area = null
+	var/last_ring
+
+/obj/structure/doorbell/Initialize()
+	if(!bell_area) // if not overridden in map, use spawn location
+		bell_area = get_area(get_turf(src))
+	last_ring = world.time
+	. = ..()
+
+/obj/structure/doorbell/attack_hand(mob/user)
+	if(world.time < last_ring + 15 SECONDS)
```

Comment:
i don't like someone being able to ring the bell every 15 seconds, at least allow people to mute it if it's spammed